test/libyul/yulOptimizerTests/redundantStoreEliminator/unrelated_relative.yul

```diff
@@ -0,0 +1,21 @@
+{
+    let c := calldataload(0)
+    mstore(c, 4)
+    mstore(add(c, 0x20), 8)
+    sstore(0, mload(c))
+    mstore(c, 9)
+    mstore(add(c, 0x20), 20)
```

Comment:
Where did these 2 `mstores` go?

Reply:
These can be removed since the execution terminates without anything reading from these locations.